WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -18,11 +18,15 @@ def index
   end
 
   COMPETITION_INFO_SERIALIZE_OPTIONS = {
-    only: %w[extra_registration_requirements enable_donations refund_policy_limit_date event_change_deadline_date waiting_list_deadline_date on_the_spot_registration on_the_spot_entry_fee_lowest_denomination qualification_results
+    only: %w[id name website start_date registration_open registration_close
+             announced_at cancelled_at end_date competitor_limit
+             extra_registration_requirements enable_donations refund_policy_limit_date event_change_deadline_date waiting_list_deadline_date on_the_spot_registration on_the_spot_entry_fee_lowest_denomination qualification_results
              event_restrictions base_entry_fee_lowest_denomination currency_code allow_registration_edits allow_registration_self_delete_after_acceptance allow_registration_without_qualification refund_policy_percent
              use_wca_registration guests_per_registration_limit venue contact force_comment_in_registration use_wca_registration external_registration_page guests_entry_fee_lowest_denomination guest_entry_status information],
-    methods: %w[registration_opened? main_event_id number_of_bookmarks using_stripe_payments? uses_qualification? uses_cutoff?],
-    include: %w[tabs],
+    methods: %w[url website short_name city venue_address venue_details latitude_degrees
+                longitude_degrees country_iso2 event_ids
+                registration_opened? main_event_id number_of_bookmarks using_stripe_payments? uses_qualification? uses_cutoff?],
+    include: %w[delegates organizers tabs],
```

Comment:
If you add all the methods for the competition controller here, then you might as well delete the humongous `COMPETITION_INFO_SERIALIZE_OPTIONS` constant from the API competitions controller

Reply:
this is the api competitions controller